include/circt/Dialect/FIRRTL/Passes.h

```diff
@@ -202,7 +202,8 @@ std::unique_ptr<mlir::Pass> createProbesToSignalsPass();
 
 std::unique_ptr<mlir::Pass> createSpecializeLayersPass();
 
-std::unique_ptr<mlir::Pass> createSpecializeOptionPass();
+std::unique_ptr<mlir::Pass> createSpecializeOptionPass(
```

Comment:
I think we can remove `std::optional`. You can use empty array as `nullopt`.

Reply:
Fixed.